status/constanst.go

```diff
@@ -0,0 +1,17 @@
+package status
```

Comment:
This file is not named correctly -- I think that you meant to call it constants. Also, why not put this in the same file as the `metrics.go` since this applies to the metrics specifically?

Reply:
Dropped the file